### DISCUSSION TITLE
Welcome to passport2-monero Discussions!

### DISCUSSION THREAD

**Discussion by mjg-foundation** [mjg-foundation/passport2-monero#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Monkeyking95** [comment]
Where do i do my donation?

**Reply by mjg-foundation** [reply]
> Where do i do my donation?

Go to the issues page, find a bounty, and comment how much you want to add to it
